Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/Play/CharacterEmoteSystem.cs

```diff
@@ -246,6 +248,13 @@ private void CleanUp(Profile profile, in DeleteEntityIntention deleteEntityInten
                 messageBus.OnPlayerRemoved(profile.UserId);
         }
 
+        [Query]
+        private void DisableCharacterController(ref CharacterController characterController, in CharacterEmoteComponent emoteComponent)
+        {
+            bool isPlayingEmote = emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE || emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE_LOOP;
```

Comment:
I see this check also in FeetIKSystem.cs. What do you think about moving it to the component level or some Utils/Extension if you pure data-only/no-logic ECS Components? That would be just `emoteComponent.IsPlayingEmote()`